src/harness/fourslashInterfaceImpl.ts

```diff
@@ -2039,3 +2043,8 @@ export interface RenameOptions {
     readonly providePrefixAndSuffixTextForRename?: boolean;
     readonly quotePreference?: "auto" | "double" | "single";
 }
+
+export interface VerifyGetImportsOptions {
+    filename: string;

```

Comment:
Is this set in stone on the VS Code side? All other properties in the protocol use `fileName`

Reply:
Nope! I just made it up. I'll change it.